guides/common/modules/proc_importing-ostree-content.adoc

```diff
@@ -0,0 +1,73 @@
+
+.Prerequisite
+* A published HTTP location (upstream URL) for the OSTree to import.
+
+.Procedure
+. In the {Project} web UI, navigate to *Content* > *Products* and click *Create Product*.
+. In the *Name* field, enter a name for your OSTree content.
+This automatically populates the *Label* field.
+. Optional: From the *GPG Key* list, select the GPG key for the product.
+. Optional: From the *SSL CA Cert* list, select the SSL CA certificate for the product.
+. Optional: From the *SSL Client Cert* list, select the SSL client certificate for the product.
+. Optional: From the *SSL Client Key* list, select the SSL client key for the product.
+. Optional: From the *Sync Plan* list, select an existing sync plan or click *Create Sync Plan* and create a sync plan for your product requirements.
+. In the *Description* field, enter a description of the product.
+. Click *Save*.
+. When the product creation completes, click *New Repository*.
+. In the *Name* field, enter a name for the repository.
+This automatically populates the *Label* field.
+. From the *Type* list, select `ostree`.
+. In the *Upstream URL* field, enter the URL of the external repository to use as a source.
+For example `http://www.example.com/rpm-ostree/`.
+. Select the *Verify SSL* check box if you want to verify that the upstream repository's SSL certificates are signed by a trusted CA.
+. In the *Upstream Username* field, enter the user name for the upstream repository if required for authentication.
+Clear this field if the repository does not require authentication.
+. In the *Upstream Password* field, enter the corresponding password for the upstream repository.
+. In the *Upstream Authentication Token* field, enter the corresponding token for the upstream repository.
+. From the *Mirroring Policy* menu, select one of the following policies to mirror OSTree content for this repository:
+* *Additive* - new content available during sync will be added to the repository, and no content will be removed.
+* *Mirror Content Only* - any new content available during sync will be added to the repository and any content removed from the upstream repository will be removed from the local repository.
+. Optional: In the *HTTP Proxy Policy* field, select an HTTP proxy.
+. Optional: Disable the *Unprotected* checkbox to require a subscription entitlement certificate for accessing the published repository.
+. Optional: In the *SSL CA Cert* field, select the SSL CA Certificate for the repository.
+. Optional: In the *SSL Client cert* field, select the SSL Client Certificate for the repository.
+. Optional: In the *SSL Client Key* field, select the SSL Client Key for the repository.
+. Click *Save*.
+. When the repository creation completes, select the new repository and click *Sync Now* to start the synchronization process.
+. To view the Synchronization Status, navigate to *Content* > *Sync Status* and expand the entry that you want to view.
```

Comment:
I am unsure about the capitalization of `Synchronization Status`. I've found it twice on the master branch all lowercase.

Reply:
I was unsure too, gonna lowercase it.